scripts/run_bitcoin_transaction.sh

```diff
@@ -5,6 +5,23 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 BASE_DIR=$SCRIPT_DIR/..
 
+# BIP activation heights (mainnet)
+BIP_16_BLOCK_HEIGHT=173805  # P2SH
+BIP_66_BLOCK_HEIGHT=363725  # Strict DER signatures
+BIP_65_BLOCK_HEIGHT=388381  # CHECKLOCKTIMEVERIFY
+BIP_112_BLOCK_HEIGHT=419328 # CHECKSEQUENCEVERIFY
+BIP_141_BLOCK_HEIGHT=481824 # SegWit
+BIP_341_BLOCK_HEIGHT=709632 # Taproot
+
+# Script flags values
+SCRIPT_BIP16=1              # (1 << 0)
+SCRIPT_VERIFY_DER_SIG=2     # (1 << 1)
+SCRIPT_VERIFY_CLTV=4        # (1 << 2)
+SCRIPT_VERIFY_CSV=8         # (1 << 3)
+SCRIPT_VERIFY_WITNESS=16    # (1 << 4)
+SCRIPT_STRICT_MULTISIG=32   # (1 << 5)
+SCRIPT_VERIFY_TAPROOT=64    # (1 << 6)
```

Comment:
I'd recommend looking here for the script flag values : https://github.com/keep-starknet-strange/shinigami/blob/d509dfeee923dd811ac299b24761f1a03b097620/packages/engine/src/flags.cairo#L52
( probably better to use [this function instead of defining the values in the bash script though](https://github.com/keep-starknet-strange/shinigami/blob/d509dfeee923dd811ac299b24761f1a03b097620/packages/engine/src/flags.cairo#L115) )

Then we can add `Flags` to the input data in the [run_raw_transaction cmd here.](https://github.com/keep-starknet-strange/shinigami/blob/d509dfeee923dd811ac299b24761f1a03b097620/packages/cmds/src/main.cairo#L209)